src/index.html

```diff
@@ -1,13 +1,62 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="device-width initial-scale=1">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <main>
+      <section class="card" data-qa="card">
+        <img
+          src="https://github.com/NadieinOleh/layout_product-cards/blob/master/src/images/imac.jpeg?raw=true"
+          alt="imac"
+          class="card__logo"
+        >
+
+        <h2 class="card__description">
+           APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <div class="card__code">
+          <p>Product code: 195434</p>
+        </div>
+
+        <div class="card__review ">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="reviews"> Reviews: 5</p>
+        </div>
+
+        <div class="card__prices">
+          <p class="card__text">Price:</p>
+          <p class="card__amount">$2,199</p>
+        </div>
+
+        <div class="card__link">
+          <a
+            class="link-buy"
```

Comment:
```suggestion
            class="card__link-buy"
```
It's not a separate BEM block, just an element of card.
Also, do you really need a div for this link?